bitcoin/src/lib.rs

```diff
@@ -29,18 +29,22 @@
 //!                            happen the implementations diverge one day.
 
 #![cfg_attr(all(not(feature = "std"), not(test)), no_std)]
+
 // Experimental features we need.
-#![cfg_attr(bench, feature(test))]
 #![cfg_attr(docsrs, feature(doc_auto_cfg))]
+#![cfg_attr(bench, feature(test))]
+
 // Coding conventions
 #![warn(missing_docs)]
+
 // Instead of littering the codebase for non-fuzzing code just globally allow.
 #![cfg_attr(fuzzing, allow(dead_code, unused_imports))]
-// Exclude clippy lints we don't think are valuable
+
+// Exclude lints we don't think are valuable
+#![allow(clippy::unconditional_recursion)] // broken; see https://github.com/rust-lang/rust-clippy/issues/12133
```

Comment:
The lint is fixed, we should remove this instead.

Reply:
Oh yeah? I tried without it and got a recursion warning, I'll double check.

Reply:
There is a false positive in the codebase still, I remove the crate level allow from all crates and added a function level allow to the callsite.

Reply:
Looks like you moved it, nor removed. Is GH confusing me?